shopelectro/models.py

```diff
@@ -82,7 +82,11 @@ class Product(
         verbose_name=_('tags'),
     )
 
-    # @todo #910:30m  Explore and doc what is vendor_code.
+    # `vendor_code` is "public id". Use this to publicly identify a product.
+    # We bring it from 1C database and sync with products.
```

Comment:
i would like to underline that `vendor_code` comes from SE stock.
Maybe you'll include these words to the comment:
```
`vendor_code` is a unique id for stock keeping unit.
It differs from site's product id and we use it to expose to the customer:
we render it at site pages, market yml offers and so on.
```
stock keeping unit (SKU) is a common term for stocked products: https://en.wikipedia.org/wiki/Stock_keeping_unit